cran-comments.md

```diff
@@ -7,15 +7,11 @@
       Status: 404
       Message: Not Found
 
-This url is valid, but it returns 404 if no Accept header is specified, which is what happens when it is automatically scanned. 
+This url is valid, but it returns 404 if no Accept header is specified, which is what happens when it is automatically scanned.
 See https://github.com/rust-lang/crates.io/issues/788 for details.
 
-*  New maintainer:
-    Ilia Kosenkov <ilia.kosenkov@outlook.com>
-  Old maintainer(s):
-    Claus O. Wilke <wilke@austin.utexas.edu>
-
-The maintainer of the package has changed.
+* Patch release is needed to synchronize templates with the new Rust 1.70 version.
+  See https://releases.rs/docs/1.70.0/ for Rust release notes and https://github.com/extendr/rextendr/pull/285 for updated templates & relevant discussion.
```

Comment:
What about something like this? My intention is:

* the comment should describe that there's a problem with the current template
* the release note of Rust 1.70 doesn't help much about what's the problem

```suggestion
* Patch release is needed to fix the bundled template incompatible with the new Rust 1.70 version, which was released on June 1st, 2023.
  See https://github.com/extendr/extendr/issues/559 for relevant discussion.
```